src/index.js

```diff
@@ -4,6 +4,9 @@ import getPropertyDescriptor from './util/get-property-descriptor';
 import debounce from 'debounce';
 import getEscapedTextContent from './util/get-escaped-text-content';
 import getCommentNodeOuterHtml from './util/get-comment-node-outer-html';
+import findSlots from './util/find-slots';
```

Comment:
I didn't want to add more code to `index.js` so I moved `findSlots()` into a module that requires code shared in `index.js`, so I moved those out, too.